file_handling/load.py

```diff
@@ -23,8 +23,6 @@ def set_sample(ws_name, geometry, material, mass_density):
     new_geometry = dict()
     for k, v in geometry.items():
         key = str(k)
-        if isinstance(v, unicode):
```

Comment:
Don't think this is required in python3

Reply:
It isn't. Also, most of mantid's objects accept `unicode` correctly in python2 so these checks can be dropped.

Reply:
Good just wanted to double check this

Reply:
Addressed in issue #12